tests/ui/never_type/eq-never-types.rs

```diff
@@ -0,0 +1,13 @@
+//@ check-pass
+//
+// Regression test for <https://github.com/rust-lang/rust/issues/120600>.
```

Comment:
this could be
`// issue: rust-lang/rust#120600`

Reply:
Is this this standardized anywhere? (this is not a normal test annotation, judging by no `@`...)

Reply:
None of the tests currently use this formatting actually....

Reply:
> Is this this standardized anywhere? (this is not a normal test annotation, judging by no `@`...)

(we don't have any standardized issue annotations yet AFAICT)

Reply:
>None


:thinking: 
`git grep "// issue" | wc -l` : 309

`git grep "// issue.*rust-lang/rust" | wc -l` : 41

afaik Jubilee planned to add a tidy lint at some point™

Reply:
M. something went wrong with my search... (possibly me)

There are a lot of different styles, but sure. Changed it.